ttnn/cpp/ttnn/operations/moreh/moreh_sgd/moreh_sgd.hpp

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/decorators.hpp"
```

Comment:
Can we please avoid device operation include here? I don’t see anything from it used here.

Reply:
resolved in [285329d](https://github.com/tenstorrent/tt-metal/pull/12378/commits/285329d75aed17e5bd826e8ee69747bdc320c677)